athena-cloudwatch-metrics/src/main/java/com/amazonaws/athena/connectors/cloudwatch/metrics/MetricUtils.java

```diff
@@ -136,18 +137,15 @@ protected static GetMetricDataRequest makeGetMetricDataRequest(ReadRecordsReques
         Split split = readRecordsRequest.getSplit();
         String serializedMetricStats = split.getProperty(MetricStatSerDe.SERIALIZED_METRIC_STATS_FIELD_NAME);
         List<MetricStat> metricStats = MetricStatSerDe.deserialize(serializedMetricStats);
-        GetMetricDataRequest dataRequest = new GetMetricDataRequest();
-        com.amazonaws.services.cloudwatch.model.Metric metric = new com.amazonaws.services.cloudwatch.model.Metric();
-        metric.setNamespace(split.getProperty(NAMESPACE_FIELD));
```

Comment:
why did these need to get deleted?

Reply:
specifically the `metric.setNamespace(split.getProperty(NAMESPACE_FIELD));`

Reply:
I don't think it had to, `metric` just isn't used anywhere.